calico/networking/configuring/bgp-to-workload.mdx

```diff
@@ -0,0 +1,302 @@
+- [Create BGPFilter resources](#configure-bgpfilter-resources)
+- [Configure parent cluster BGP topology](#configure-parent-cluster-bgp-topology)
+- [Configure BGP peering with workloads](#configure-bgp-peering-with-workloads)
+- [Configure more than one nested cluster](#configure-more-than-one-nested-cluster)
+
+### Configure global BGP settings
+
+In order to ease configuration of the `BGPPeer` resources in the nested clusters,
+the workload BGP peering feature uses a single virtual IP (per IP version) that
+is provisioned on every node in the parent cluster. The workloads are configured
+to peer with this virtual IP rather than needing to know the specific IP of their
+parent node.
+
+To avoid conflicts with your network, the addresses are configurable through
+the `BGPConfiguration` resource and there are no defaults. They must be
+configured to enable the feature:
+
+- For IPv4, we recommend using a link-local IP address that is not otherwise
+  in use on the parent cluster's network. Note that $[prodname] uses
+  `169.254.0.1` and `169.254.0.2` internally so these must not be used. You may also
+  wish to avoid cloud provider metadata addresses, such as `169.254.169.254`
+  (as used by AWS). In this guide we'll use `169.254.0.179`; you can use the same
+  address unless it conflicts with some other use on your nodes.
+
+- If using the IPv6 address, you _must not_ use a link local address (because
+  IPv6 requires link-local addresses to be qualified with a scope tag, and $[prodname]
+  currently ahs no way to provision the correct scopes). Instead, we recommend
+  using a ULA address that is not otherwise in use in your network. In this
+  guide we'll use `fdc9:9723:09bc::1`(which was chosen randomly according to
+  RFC 4193 for this guide, so it may be suitable for your use as well).
+
+Once you have chosen the IP address(es), configure the BGPConfiguration resource.
+(You can omit the IPv6 address if not using IPv6.)
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: BGPConfiguration
+metadata:
+ name: default
+spec:
+  ...
+  localWorkloadPeeringIPV4: "169.254.0.179"
+  localWorkloadPeeringIPV6: "fdc9:9723:09bc::1"
+```
+
+### Create BGPFilter resources
+
+Before creating the BGPPeer to peer with the workloads, we create
+`BGPFilter` resources to control which routes are imported and exported.
+
+- Because the workloads are running on the $[prodname] cluster, we can assume that
+  they already have a valid default route via their host node. Hence, we don't
+  need to export any routes to the workloads. Create the following `BGPFilter`,
+  which can be used to block export of any routes.
+
+  ```yaml
+  apiVersion: projectcalico.org/v3
+  kind: BGPFilter
+  metadata:
+    name: no-export
+  spec:
+    exportV4:
+      - action: Reject
+    exportV6:
+      - action: Reject
+  ```
+
+- For added security, it is also wise to limit what routes we'll accept from
+  the workloads. Create a `BGPFilter` that only accepts routes that are within
```

Comment:
Again, either modify a noun with the code name (`BGPFilter` resource), or describe the actual thing (BGP filter). Same idea throughout the doc.